lib/trino-hdfs/src/main/java/io/trino/filesystem/hdfs/HdfsFileSystemManager.java

```diff
@@ -61,9 +61,9 @@ public HdfsFileSystemManager(
         modules.add(new HdfsModule());
         modules.add(new HdfsAuthenticationModule());
         modules.add(new HiveCosModule());
+        modules.add(new CatalogNameModule(catalogName));
         modules.add(binder -> {
             binder.bind(OpenTelemetry.class).toInstance(openTelemetry);
-            binder.bind(CatalogName.class).toInstance(new CatalogName(catalogName));
```

Comment:
this seems not relates to the change?

Reply:
Not strictly related, yes. This is a follow-up cleanup.